mutt/file.c

```diff
@@ -1324,6 +1329,8 @@ int mutt_file_to_absolute_path(char *path, const char *reference)
 
   dirpath = mutt_file_dirname(reference);
   mutt_str_strfcpy(abs_path, dirpath, PATH_MAX);
+  /*mutt_file_dirname malloc char *dirpath, so here free it*/
+  free(dirpath);
```

Comment:
Please use our macro `FREE()` which takes a double pointer.
It sets the variable to NULL, too.